handlers/tokeninfo/jwt/tokeninfo.go

```diff
@@ -164,10 +166,14 @@ func claimAsInt64(t *jwt.Token, claim string) (int64, bool) {
 }
 
 func getClaim(t *jwt.Token, claim string) (interface{}, bool) {
-	c, has := t.Claims[claim]
-	if !has {
-		log.Printf("Missing claim %q for token %v", claim, t.Raw)
-		return "", false
+	if claims, ok := t.Claims.(jwt.MapClaims); ok {
```

Comment:
@jeffsantarossa, see https://stackoverflow.com/questions/30162256/convert-nil-interface-to-pointer-of-something-in-go-lang

"The value of ok is true if the assertion holds. Otherwise it is false and the value of v is the zero value for type T. No run-time panic occurs in this case."